generic/denials/deny-general-account-deletion-request.md

```diff
@@ -1 +1 @@
-Sorry, but we do not currently delete accounts unless certain prerequisites are met (as per https://osu.ppy.sh/legal/privacy#your-rights-and-control). If you wish to disable access to your own account, you may change your password to a random string of letters and numbers in your [profile settings](https://osu.ppy.sh/forum/ucp.php?i=profile&mode=reg_details).
\ No newline at end of file
+Sorry, but we do not currently delete accounts unless certain prerequisites are met (as per [your rights and control](https://osu.ppy.sh/legal/privacy#your-rights-and-control)). If you wish to disable access to your own account, you may change your password to a random string of letters and numbers in your [profile settings](https://osu.ppy.sh/forum/ucp.php?i=profile&mode=reg_details).
```

Comment:
use `https://osu.ppy.sh/home/account/edit` would be better

Reply:
Fixed